subsys/bluetooth/controller/ll_sw/nordic/lll/lll_df_types.h

```diff
@@ -88,7 +94,7 @@ struct node_rx_iq_report {
  */
 struct lll_df_sync_cfg {
 	uint8_t is_enabled:1;
-	uint8_t slot_durations:2; /* One of possible values: 1us, 2us. */
+	uint8_t slot_durations:2; /* Bit field where: BIT(1) is 1us, BIT(2) is 2us. */
```

Comment:
```suggestion
	uint8_t slot_durations:2; /* Bit field where: BIT(0) is 1us, BIT(1) is 2us. */
```

Are there 3 values hence need 2 bits? or is it bitmask where both 1us and 2us is allowed? If only one of 1us or 2us is allow why not use 1-bit field?

Reply:
BT 5.3 Core spec defines slot durations as: `0x01 Switching and sampling slots are 1 μs each` and `0x02 Switching and sampling slots are 2 μs each`. To avoid translation of the values (and extra operations) I decided to use one more bit and stick to definitions and assigned values. This also avoids confusion when reading the code executed from host down to LLL. 

BTW. Even if I change it to use single bit, the size of the struct will be the same.

Reply:
Vinayak is still correct that if you use `BIT(1)` and `BIT(2)`, then we need `uint8_t slot_durations:2` as `BIT(2)` would set the 3rd bit (value 4).

Reply:
Ah, I've omitted it, thanks!